statuts.md

```diff
@@ -143,16 +144,19 @@ Il préside les réunions du conseil d’administration
 et des assemblées générales, dont il décide l’ordre
 du jour. Il présente le bilan moral de l’association
 à l’assemblée générale une fois par an. Il peut occasionnellement
-déléguer ses pouvoirs à un autre
-membre du conseil d’administration.
+déléguer ses pouvoirs au vice-président ou à un autre
```

Comment:
Est-ce vraiment le cas ? Le vice-président, pour moi, est censé avoir les pleins pouvoirs uniquement si le président n'est pas disponible.

Or c'est déjà le cas. Plus loin il est noté qu'en cas d'absence prolongé du président il récupère tous les pouvoirs. 

Cette partie ne concerne que **les absences occasionnel** ! En pratique ça donne un pouvoir au président, celui d'empecher le déroulement d'une réunion de CA par exemple si il n'est pas là. Celles-ci devant être signé par lui, si il ne délègue pas son pouvoir, la réunion ne peut avoir lieu. Mais je ne trouve pas ça déconnant, pour les **empêchements occasionnels** le président puisse faire repousser une réunion ou qu'on prenne des décisions qui peuvent l'engager (il est le représentant légal de l'asso).

A partir du moment que cette partie ne concerne que les absences occasionnels je ne voie pas de problèmes.

Je pensais compléter ça avec un paragraphe dans le règlement intérieur qui précise que si un membre du bureau ne répond pas pendant 15 jours (sans que ce soit prévu) il soit considéré comme en absence prolongé.